src/indexer.rs

```diff
@@ -1,14 +1,17 @@
 use std::{str::FromStr, time::Instant};
 
 use log::info;
-use reth_primitives::{Address, BlockHash, Bloom, Header, Log, TransactionSignedNoHash, H256};
+use reth_primitives::{Address, BlockHash, Bloom, Header, Log, H256, TransactionSigned};
```

Comment:
use to get `TransactionSignedNoHash` now get `TransactionSigned ` not sure if its just a type change but use to use 

[ ](https://github.com/joshstevens19/reth-indexer/pull/15/files#diff-ddbcb943b19f1da0537579689759cf2e4000002a95c902c425938d026600ee5eL123)

now

[
](https://github.com/joshstevens19/reth-indexer/pull/15/files#diff-ac2ac359957203dd5b34f40737b4580ff70e166d887226350d589f985f86a375R41)

@mattsse